templates/glossary.md

```diff
@@ -134,9 +137,27 @@ Examples of such commands are the `def`, `theorem`, `constant` or `example` comm
 
 Further detail can be found in the [Lean documentation](https://leanprover.github.io/lean4/doc/declarations.html#basic-declarations).
 
+### dependent type theory
+
+A [type theory](#type-theory) in which one additionally may have types which depend on a parameter, such as the type of "days of a calendar month", whose specific values depend on the particular month, given that different months have different numbers of days.
+Further examples may be found in the resources below.
+Lean's implementation of dependent type theory is based on what is known as the *Calculus of Constructions*, allowing for its use both in complex mathematical reasoning as well as software verification.
+
+##### See Also
+
+* [Section 2 of Theorem Proving in Lean](https://leanprover.github.io/theorem_proving_in_lean/dependent_type_theory.html), which discusses Lean's specific version of dependent type theory
+
+* [Calculus of Constructions, from Wikipedia](https://en.wikipedia.org/wiki/Calculus_of_constructions), for a further overview of the calculus of constructions
+
+* [Dependent Type Theory from nLab](https://ncatlab.org/nlab/show/dependent+type+theory), for a general treatment of dependent type theories
+
+* [Mike Shulman's In Praise of Dependent Types](https://golem.ph.utexas.edu/category/2010/03/in_praise_of_dependent_types.html)
+
+* [Andrej Bauer's answer to "What makes dependent type theory more suitable than set theory for proof assistants?"](https://mathoverflow.net/a/376973)
+
 ### diamond
 
-The existence of multiple conflicting [terms](#term) of a [class](#class) found within the typeclass [instance](#instance) graph.
+The existence of multiple conflicting [terms](#term) of a [class](#class) found within the [typeclass](#class) [instance](#instance) graph.
```

Comment:
`#class` is already linked in this sentence and with this change there are two different links right next to each other, which can look confusing -- our style sheet doesn't underline links. Do you think this is necessary?

Reply:
Definitely a good point, reverting.